### ISSUE TITLE
Advice on how to create one additional level above bip44?

### ISSUE THREAD

**Issue by thinkloop** [iancoleman/bip39#74]
Hey Ian, the tools you've built are awesome and invaluable to the community, thank you very much for all your effort!

I was wondering if I could ask some advice. 

I am managing btc and alts for a bunch of friends. I would like for each of them to have their own bip44 wallet, however I would also like to have a single super-master key that can access all funds. Something like:

`m / super_account' (i.e. bob's wallet) / purpose' / coin_type' / ...`

Instead of:

`m / purpose' / coint_type' / ...`

Basically a master xprv/xpub that has as children bip44 xprv/xpub. Something like `m/0'/44'/0'/0'/0` instead of `m/44'/0'/0'/0`.

Any suggestions on how to go about that and whether it makes sense?

**Comment by iancoleman** [comment]
First advice is don't manage crypto for friends!

But yes, what you propose can be done.

The technique you propose makes sense if you want to give your friends a watch-only bip44 wallet which is derived from a single master mnemonic you control.

Be aware the xpub to give for a bip44 watch-only wallet is usually the Account Extended Public Key.

From your post it seems you've understood everything required to go about doing this, so if you need more clarification I am happy to answer any more specific questions you may have.

One last thing I suggest is to keep meticulous documentation for this sort of stuff, and keep that documentation secure. Standards exist for a reason, and doing custom stuff comes with inherent risk. I would personally just go with multiple standard bip39 / bip44 accounts.

**Comment by thinkloop** [comment]
> First advice is don't manage crypto for friends!

haha, they're literally forcing me to! :-D They have no interest in learning how to trade, find liquidity, manage wallets, etc., they just want that sweet crypto.

> I suggest to keep meticulous documentation for this sort of stuff, and keep that documentation secure. Standards exist for a reason, and doing custom stuff comes with inherent risk. I would personally just go with multiple standard bip39 / bip44 accounts.

Thank you, absolutely, part of the reason I am approaching it this way is to minimize any possible confusion. At the end every person should have a complete, proper bip44, I just want one level above so I can trade/spend on their behalf.

> From your post it seems you've understood everything required to go about doing this, so if you need more clarification I am happy to answer any more specific questions you may have.

Using [your tool]([https://iancoleman.github.io/bip39/) if I provide `m/0'/44'/0'/0'/0` for the `BIP32 Derivation Path` I won't be able to get the xprv/xpub for each bip44 child without doing a tiny bit of coding to calculate it, correct? Phrased differently, the xprv/xpub that's output is for the super master key that I would keep and use, but not the xpub I could give to them to watch their wallet, correct?

**Comment by iancoleman** [comment]
Steps I would take to do this:

* generate random mnemonic to be the super root mnemonic (and store it very securely)
* generate a Bip32 Root Key for friend 0
    * click bip32 tab
    * enter derivation path `m/0'`
    * copy the bip32 extended private key - this is the Bip32 Root Key for friend 0 - you don't want to record this or give this to anyone, but you do need to temporarily use it for a short while in the following steps.
* refresh the page to clear the super root mnemonic
* paste the xprv key into the Bip32 Root Key field
* give friend 0 the Account Extended Public Key from the bip44 tab - friend 0 can import this into most bip44 compatible wallets (eg mycelium / electrum) to watch their funds.
* repeat for each friend using the original super root mnemonic and derivation path `m/X'`
* at the end, copy some meaningless text into your clipboard and take any other precautions necessary (close the tab, reboot computer, etc).

To reiterate, I would personally use multiple bip39 / bip44 accounts for each user. The record-keeping requirements are almost the same, but the risk is much lower.

**Comment by thinkloop** [comment]
Works perfectly, thank you!

> To reiterate, I would personally use multiple bip39 / bip44 accounts for each user. The record-keeping requirements are almost the same, but the risk is much lower.

If I have a dozen accounts, are you suggesting that it is safer/smarter to generate and manage 12 paper pneumonics that are standards-compliant, rather than relying on this custom process that allows me to have 1 paper to manage?

**Comment by iancoleman** [comment]
> If I have a dozen accounts, are you suggesting that it is safer/smarter to generate and manage 12 paper pneumonics that are standards-compliant, rather than relying on a custom process that allows for 1 paper to manage?

Yes, but that's me.

On the flip side, the discussion in this issue has shown that if the custom derivation scheme is desired then it will work as intended.

I tend to design things with end-of-life high on the priority list, and the handover process for custom stuff is a poor compared with a standards compliant process. Your priorities are likely to be different to mine, so trust yourself to go with what works best for you!
